### PR TITLE
Update note on definitions in mmset.html

### PR DESCRIPTION
**Pull request by jkingdon**
- Remove some commented out markup
- Link to ~ conventions which has a specification for the set.mm definition checker
- Remove paragraph which contains an incomplete version of that specification.

### CODE REVIEW

File: mmset.raw.html [hunk 1]
```diff
@@ -4304,7 +4298,9 @@
 <P>
 For the specific case of the set theory database set.mm, Mario Carneiro added
 an enhancement to the <A HREF="../index.html#mmj2">mmj2</A> program that will
-verify the soundness of all but four definitions.  This test can be turned on
+verify the soundness of all but four definitions.  There is a specification
+for this definition checker in the "additional rules for definitions" section
+of the ~ conventions page. The mmj2 definition checker can be turned on
```
Comment (benjub) [inline mmset.raw.html:4303]: That page uses the convention "double space after period ending a sentence".

File: set.mm [hunk 2]
```diff
@@ -420110,6 +420110,8 @@ varying definitions (some start from 0, others start from 1), but
        metatheorem that underlies this.
        </p>
 
+       <p><b>Additional rules for definitions</b></p>
+
```
Comment (benjub) [inline set.mm:420114]: That makes it easier to locate, thanks.

File: mmset.raw.html [hunk 3]
```diff
@@ -4405,23 +4401,6 @@
...
-possible renaming of bound variables).  The soundness of such definitions is
-simple to check by inspection:  if (1) the left-hand constant symbol is new and
-doesn't appear in the right-hand side, (2) all variables are distinct (all of
-them appear in a single $d statement), and (3) we can prove (with a Metamath
-theorem, e.g. ~ pwjust for ~ df-pw ) that the right-hand expression equals
-itself with all variables renamed, then the definition is sound.  This
-convention allows all but four definitions to be verified automatically by mmj2
-as described above.
-</P>
-
```
Comment (benjub) [inline mmset.raw.html:4424]: I agree: subsumed by the description on the ~conventions page

**Review by benjub — CHANGES_REQUESTED** [review]
Just one "double space" to add and it's good.

File: mmset.raw.html (hunk 1 above)
Comment (jkingdon) [reply mmset.raw.html]: Ah, good catch. Fixed.

**Review by benjub — APPROVED** [review]

**Pull request merged**